src/osd/scrubber/scrub_backend.cc

```diff
@@ -1241,7 +1241,7 @@ bool ScrubBackend::compare_obj_details(pg_shard_t auth_shard,
     auth_bl.push_back(auth_attr->second);
 
     if (!can_bl.contents_equal(auth_bl)) {
-      format_to(std::back_inserter(out), "{}object info inconsistent ", sep(error));
+      fmt::format_to(std::back_inserter(out), "{}object info inconsistent ", sep(error));
```

Comment:
line too long
```suggestion
      fmt::format_to(std::back_inserter(out), "{}object info inconsistent ",
                     sep(error));
```

Reply:
... and similar instances below, I'm counting 6 in total.

Reply:
Fixed